docs/deployment/setup_gardener.md

```diff
@@ -1,44 +1,412 @@
+
+Assuming the base domain for a Gardener landscape is `gardener.crazy-botany.cloud`, the following example shows a possible way to structure DNS records:
+
+| Component                | General                                            | Example                                              |
+|:-------------------------|:---------------------------------------------------|:-----------------------------------------------------|
+| Gardener Base            | gardener.\<showroom-base-domain\>                  | gardener.crazy-botany.cloud                          |
+| Virtual Garden           | api.garden.\<gardener-base-domain\>                | api.garden.gardener.crazy-botany.cloud               |
+| Gardener Ingress         | ingress.garden\<gardener-base-domain\>             | ingress.garden.gardener.crazy-botany.cloud           |
+| Gardener Dashboard       | dashboard.ingress.\<gardener-base-domain\>         | dashboard.ingress.garden.gardener.crazy-botany.cloud |
+| Shoot Internal           | internal.\<gardener-base-domain\>                  | internal.gardener.crazy-botany.cloud                 |
+| Shoot (default) External | shoot.\<gardener-base-domain\>                     | shoot.gardener.crazy-botany.cloud                    |
+| Unmanaged Seed Ingress   | ingress.soil.\<gardener-base-domain\>              | ingress.soil.gardener.crazy-botany.cloud             |
+| Managed Seed             | \<seed-name>.seed.\<gardener-base-domain\>         | cc-ha.seed.gardener.crazy-botany.cloud               |
+| Managed Seed Ingress     | ingress.\<seed-name>.seed.\<gardener-base-domain\> | ingress.cc-ha.seed.gardener.crazy-botany.cloud       |
+
+Reference documentation:
+
+- [Gardener Configuration](https://github.com/gardener/gardener/blob/master/docs/operations/configuration.md#system-configuration)
+- [DNS records](https://github.com/gardener/gardener/blob/master/docs/extensions/resources/dnsrecord.md)
+
+### Backup Bucket
+
+The virtual Garden cluster's `etcd` is governed by [etcd-druid](https://github.com/gardener/etcd-druid). Backups are created continuously and stored to a supported object store. Credentials to manage an object store on the chosen infrastructure are required.
+
+### Infrastructure
+
+For each infrastructure supported by Gardener a so-called `provider-extension` exists. It implements the required interfaces. For this example, the [openstack extension](https://github.com/gardener/gardener-extension-provider-openstack) will be used.
+
+Access to the respective infrastructure is required and needs to be handed over to Gardener in the provider-specific format (e.g., Openstack application credentials).
+
+Reference documentation:
+
+- [Gardener extensibility](https://github.com/gardener/gardener/tree/master/docs#extensions)
+- [Provider openstack](https://github.com/gardener/gardener-extension-provider-openstack/tree/master/docs)
```

Comment:
Can you use relative links here?

Reply:
makes sense :) thanks! Done throughout the document.